src/pgduckdb_types.cpp

```diff
@@ -571,6 +571,7 @@ GetPostgresDuckDBType(duckdb::LogicalType type) {
 	case duckdb::LogicalTypeId::INTEGER:
 		return INT4OID;
 	case duckdb::LogicalTypeId::BIGINT:
+	case duckdb::LogicalTypeId::UBIGINT:
```

Comment:
I don't think this is correct. All other unsigned ints use a type that is one size larger than their own type, which makes sense. The largest half of an unsigned int does not fit into a signed int of the same size.

We might want to start depending on `pguint`, to provide actual unsigned integer types: https://github.com/petere/pguint

Reply:
ubigint is required as `manifest_sequence_number` field returns that from duckdb execution - so we don't match it and execution fails. i can add comment for this in code.

Reply:
Hmm, that's pretty annoying. Now I understand why you added it to this PR. 

The main problem I have is that right now we fail hard for ubigint, which seems nicer than to silently convert too-high ubigints to negative numbers. 

How about we just don't return that field for now? i.e. make the type:
```sql
CREATE TYPE iceberg_metadata_record AS (
  manifest_path TEXT,
-- manifest_sequence_number is not included, because it is a UBIGINT, which we don't support yet.
  manifest_content  TEXT,
  status TEXT,
  content TEXT,
  file_path TEXT
);
``

And then in another PR we can support UBIGINT, probably by converting it into a numeric instead of a BIGINT.

Reply:
I think that will work for now